CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
+- Tarantool benchmark tool.
```

Comment:
`- Tarantool benchmark tool (early alpha, API can be changed in the near future).`